german/the-black-hack.md

```diff
@@ -113,7 +113,7 @@ Zeit im Spiel wird vor allem auf zwei Arten nachgehalten - **Momente** (Runden)
 
 ## Bewegung und Entfernung
 
-Anstatt genaue Werte nachzuhalten, benutzt TBH abstrakte Bereiche, mit denen die Entfernung dargestellt wird. **Benachbart**, **in der Nähe**, **abseits** und **entfernt**. Während eines Zugs können sich Charaktere irgendwo **in der Nähe** bewegen und ihre Handlung jederzeit während der Bewegung durchführen. Sie können ihre Handlung verfallen lassen, um sich **abseits** zu positionieren. Alles jenseits von **abseits** gilt als **entfernt** und würde drei Bewegungen erfordern.
+Anstatt genaue Werte nachzuhalten, benutzt TBH abstrakte Bereiche, mit denen die Entfernung dargestellt wird. **Dicht**, **Nah**, **Weit** und **Fern**. Während eines Zugs können sich Charaktere irgendwo **in der Nähe** bewegen und ihre Handlung jederzeit während der Bewegung durchführen. Sie können ihre Handlung verfallen lassen, um sich **abseits** zu positionieren. Alles jenseits von **abseits** gilt als **entfernt** und würde drei Bewegungen erfordern.
```

Comment:
Dicht, Weit, Fern sind Ideen von Marc, das Nah habe ich eingefügt weil es imho gut in die Reihe passt.